scripts/verify_adv_IM.py

```diff
@@ -6,13 +6,47 @@
 
```

Comment:
Would rename this script from verify to something else. Verify implies verification - rather than completeness checking. But I guess we do have hangovers like validateVM etc?

Reply:
the initial name was test_adv_im_status.py but it is triggering out automated testing ._.
so i had to find a word that is closer but not 'test'

Reply:
:laughing: That's perfect. I can't think of a better name / descriptor off the top of my head.

Reply:
check_adv_im.py? Otherwise it's fine for now, and we can change it later if there're issues

Reply:
i've moved the script to IM_calc repo, and renamed it check_adv_IM_status.py

Reply:
Perfect. That works for me!